client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.tsx

```diff
@@ -57,6 +57,10 @@ export function getEnhancedTasks(
 
 	const translatedPlanName = productSlug ? PLANS_LIST[ productSlug ].getTitle() : '';
 
+	const firstPostPublished =
+		Boolean( tasks?.find( ( task ) => task.id === 'first_post_published' )?.completed ) ||
+		! isStartWritingFlow( flow );
```

Comment:
Do we need the check for `! isStartWritingFlow( flow )` here or can we do without it, like this?
```suggestion
	const firstPostPublished = Boolean(
		tasks?.find( ( task ) => task.id === 'first_post_published' )?.completed
	);
```

Reply:
You're right, we can remove it :)